### DISCUSSION TITLE
[rejected] Filtrowanie ofert po produkcie dla API publicznego

### DISCUSSION THREAD

**Discussion by kormichu** [allegro/allegro-api#4329] [Suggestions - rejected]
Cześć,

Czy da się / dałoby się wyfiltrować oferty po identyfikatorze produktu ? Z tego co widzę /offers/listing nie ma takiej możliwości: https://developer.allegro.pl/documentation/#section/Authentication

**Comment by MaciejFrackowiak** [comment]
Zgadza się, [GET /offers/listing](https://developer.allegro.pl/documentation/#operation/getListing) nie zwraca product.id dla ofert z przypisanym produktem. Dla wyszukiwania produktów natomiast istnieje osobny zasób - [GET /sale/products](https://developer.allegro.pl/documentation/#operation/getSaleProducts).

**Comment by kormichu** [comment]
To już nawet nie chodzi o to że GET /offers/listing nie zwraca product.id choć dobrze by było wprowadzić coś takiego. Bardziej chodzi mi o możliwość wyfiltrowania ofert na poziomie samego requestu tak jak teraz można to zrobić przy pomocy parametru phrase dla nazwy. Problem w tym że nie jest to dokładne. Chciałbym odtworzyć funkcjonalność, która jest dostępna u Was na interfejsie użytkownika:

https://allegro.pl/oferta/apple-iphone-x-oryginal-gwarancja-8903955225

Konkretnie chodzi o boxa z innymi ofertami tego produktu "Najtańsze od", "Najszybciej od". Mogę co prawda wejść na adres:

https://allegro.pl/produkt/<identyfikator produktu>

i jakoś pobrać sobie ze źródła strony to co chce, ale to trochę obejście problemu na około

**Comment by MaciejFrackowiak** [comment]
Dziękuję za doprecyzowanie - Twoją sugestię dodałem do listy sugestii dot. dalszego rozwoju REST API.

**Comment by AureliuszBrussy** [comment]
Nie planujemy rozszerzać zakresu danych zwracanych w [GET /offers/listing](https://developer.allegro.pl/documentation/#operation/getListing).
